modes/intake.md

```
@@ -0,0 +1,109 @@
+2. Record **only the sources that were actually merged**, so the next run
+   proposes only new material:
+
+```bash
+node intake.mjs --commit <path> [<path> …]   # the confirmed sources
+node intake.mjs --commit                     # only if ALL were merged
+```
+
+   Never blanket-commit after a partial confirmation — a declined source
+   must stay `new` so it is re-proposed next time.
+
+3. Verify: `node doctor.mjs` should report the profile prerequisites
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)